srcpkgs/trident-icons/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'trident-icons'
+pkgname=trident-icons
+version=2019.10.12
+revision=1
+short_desc="Icon themes from Project Trident"
+maintainer="Ken Moore <ken@project-trident.org>"
+license="MIT"
```

Comment:
MIT or GPL3?

Reply:
MIT is correct for this package.

Reply:
[This](https://github.com/project-trident/trident-icons/blob/master/le-trident/COPYING) is what is confusing.

Reply:
Ah, interesting. The guy who set this up tagged it all as MIT in the base of the repo. Let me see if there is some way to dual-tag the package as both MIT and GPL3, since it looks like a combination of some GPL3 icons + a bunch of MIT ones.

Reply:
Fixed! I double checked that the underlying icons are just GPL3 and not GPL3+ as well.